ext/gd/libgd/gd.c

```diff
@@ -3108,7 +3108,7 @@ int gdImagePaletteToTrueColor(gdImagePtr src)
 		const unsigned int sy = gdImageSY(src);
 		const unsigned int sx = gdImageSX(src);
 
-		src->tpixels = (int **) gdMalloc(sizeof(int *) * sy);
+		src->tpixels = (int **) gdCalloc(sizeof(int *), sy);
```

Comment:
Please add a comment that this is an intended change, so it won't be inadvertently dropped when syncing with upstream.